### DISCUSSION TITLE
Vant 默认版本已切换至 3.x（适配 Vue 3）

### DISCUSSION THREAD

**Discussion by chenjiahan** [youzan/vant#10271] [Show and tell]
目前 [Vue 3 现已成为新的默认版本](https://blog.vuejs.org/posts/vue-3-as-the-new-default.html)，因此 Vant 也进行了相应的调整，包括：

### npm tag 变更

- npm 上的 `latest` tag 默认指向 Vant 3（对应 Vue 3）
- npm 上的 `latest-v2` tag 会指向 Vant 2 的最新版（对应 Vue 2）

### 文档网站变更

- 文档网站现在默认展示 Vant 3 的文档：https://vant-contrib.gitee.io/vant/
- Vant 2 的文档地址迁移至：https://vant-contrib.gitee.io/vant/v2

如果大家在对上述版本切换有任何问题，欢迎在此帖下进行反馈 ^_^

**Comment by kevinzhou1990** [comment]
vue 3 使用官网 webpack5+vue3+vant+SSR的方案，全局引入vant，服务端打包的结果，出现了window之类的内容，使用SSR，vant只能按需引入组件，不能全局引入所有组件吗？

**Reply by chenjiahan** [reply]
可以全局引入的，对于这个问题，欢迎创建一个 issue 来反馈
